src/utils/resources/vmi/utils/credentials.ts

```diff
@@ -0,0 +1,18 @@
+import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
```

Comment:
please name this file with a better meaning, such as cloud-init.ts , its a share util file, so everything belong to cloud-int can rest here.